nevergrad/functions/olympussurfaces/core.py

```diff
@@ -0,0 +1,121 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Based on https://github.com/aspuru-guzik-group/olympus
+
+import numpy as np
+from functools import partial
+from nevergrad.parametrization import parameter as p
+from ..base import ExperimentFunction
+
+import nevergrad as ng
+
+try:
+    from olympus import surfaces  # type: ignore
+    from olympus import noises  # type: ignore
+    from olympus import Emulator  # type: ignore
+    from olympus.datasets import Dataset  # type: ignore
+except ImportError as e:
+    raise ng.errors.UnsupportedExperiment("Please install olympus for Olympus experiments") from e
```

Comment:
This needs to be done inline in the function, otherwise it will break all other experiments.

Also, instead of all the type ignore, add `olympus` to mypy.ini:
https://github.com/facebookresearch/nevergrad/blob/603221a535dfaebfd7d76b56732cbd9b716292eb/mypy.ini#L3